Source/Core/Core/HW/DSP.cpp

```diff
@@ -212,6 +212,7 @@ void EnableInstantDMA()
 	CoreTiming::RemoveEvent(et_CompleteARAM);
 	CompleteARAM(0, 0);
 	instant_dma = true;
+	ERROR_LOG(DSPINTERFACE, "Enabling Instant ARAM DMA hack");
```

Comment:
In the context of the logger, an error is a condition that should never happen, or signifies that something could not be recovered from – critical/exceptional cases, etc. Not, "this is a codepath that shouldn't exist", otherwise we may as well add that to a pile of other codepaths as well. Then, after, we can consider removing the error log class altogether, because at that point, it has lost its meaning. It's effectively become the notice log class.